.github/actions/private-tangle/setup/action.yml

```diff
@@ -1,47 +1,36 @@
-name: 'private-tangle-setup'
-description: 'Setup a private tangle'
+name: "private-tangle-setup"
+description: "Setup a private tangle"
 runs:
   using: "composite"
   steps:
     - name: Clone private tangle files
       uses: actions/checkout@v3
       with:
-        repository: iotaledger/hornet
-        path: hornet
+        repository: iotaledger/iota-core
+        path: iota-core
+
+    - name: Prepare files for start and stop
+      shell: bash
+      run: |
+        echo "docker compose down -v" >> cleanup.sh
+        echo "rm *.snapshot" >> cleanup.sh
+        chmod +x cleanup.sh
+
+        sed -i -n -e :a -e '1,5!{P;N;D;};N;ba' run.sh
```

Comment:
Alright :trollface:

Reply:
It just removes the last 5 lines of the file (because otherwise it doesn't run the private tangle detached), there is probably a cleaner way, but I didn't want to spend more time on it since its working fine :trollface:

Reply:
still maybe add a comment what this command exactly does?

Reply:
Then it should be easy to notice, because it will fail :trollface: 
But will add a comment :+1:

Reply:
Done